docs/docs/mdx/getting-started.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Getting started
```

Comment:
Since I just finished reading the Gatsby styleguide, I can say that this should be in Title Case 😄

Reply:
I definitely debated that choice because I like the casualness of lowercase. But yeah, trying to make these enterprise ready ;)